### DISCUSSION TITLE
nullishCoalescingOperator not enabled....

### DISCUSSION THREAD

**Discussion by kscottthomas** [KevinVandy/material-react-table#197] [Q&A] [answered]
I am stumped, I am getting this message even though I have installed the plugin and enabled it in my package.json  
node_modules\@tanstack\match-sorter-utils\build\umd\index.production.js: Support for the experimental syntax 'nullishCoalescingOperator' isn't currently enabled. 

end of my package.json

"babel": {
    "presets": [
      "react-app"
    ],
    "plugins": [
      "@babel/plugin-proposal-nullish-coalescing-operator"
    ]
  },

Thank you in advance!

**Comment by KevinVandy** [comment]
TanStack Table itself recently introduced a bug where it is not compatible with webpack before version 5. I'm looking into making a PR upstream in that project to fix it.

**Reply by KevinVandy** [reply]
@kscottthomas  I actually don't know when this will get fixed. For some, simply upgrading their `react-scripts` package to the latest version or making sure they are using webpack v5 fixes the problem

**Comment by KevinVandy** [comment, marked as answer]
@kscottthomas I think this should be fixed in v1.3.13. TanStack Table just got a ESM and webpack config overhaul
